media/css/firefox/new/browse-up-to-you.less

```diff
@@ -0,0 +1,54 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../sandstone/lib.less';
+
+// generates a < tablet media query for all sections (except intro)
+.common-mobile-props(@padding: 250px) {
+    @media only screen and (max-width: @breakTablet) {
+        background-position: bottom 20px center;
+        min-height: 0;
+        padding-bottom: @padding;
+
+        .copy {
+            float: none;
+            width: auto;
+        }
+    }
+}
```

Comment:
I don't see this mixin used anywhere. Safe to remove it from all these `.less` files?